src/mscorlib/src/System/Reflection/Emit/ILGenerator.cs

```diff
@@ -1174,6 +1174,25 @@ public virtual void ThrowException(Type excType)
             this.Emit(OpCodes.Throw);
         }
 
+        private static Type GetConsoleType()
+        {
+#if PLATFORM_UNIX
+            Type t = null;
+            try
+            {
+                t = Type.GetType("System.Console, System.Console, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a", throwOnError: false);
```

Comment:
That's part of why it's a hack ;)  Our tests will start to fail, which is how we discovered this wasn't working the first place.  I don't know if we have another mechanism available.  Like I said, it's a hack; I don't love the solution, I'm just not aware of a significantly better solution.